src/spicetools/tests/test_timeutil.py

```diff
@@ -0,0 +1,42 @@
+import ctypes
+
+import numpy as np
+import pytest
+
+from spicetools.timeutil import times2et
+from spicetools.kernelutil import make_meta
+import spiceypy as sp
+import os
+
+FILES = [
+    "$KERNELS/lsk/naif0012.tls",
+    "$KERNELS/pck/gm_de440.tpc",
+    "$KERNELS/pck/pck00011.tpc",
+]
+TEST_META_FILE = "test_meta.mk"
```

Comment:
import tempfile

temp_dir = tempfile.TemporaryDirectory()
TEST_META_FILE = os.path.join(temp_dir.name,"test_meta.mk")
# instead of os.remove(TEST_META_FILE)
temp_dir.cleanup()

Reply:
Try to use a pytest fixture to perform a setup and teardown operation 

[https://pytest-with-eric.com/pytest-best-practices/pytest-setup-teardown/](https://pytest-with-eric.com/pytest-best-practices/pytest-setup-teardown/)